allennlp/tests/training/trainer_test.py

```diff
@@ -121,6 +122,26 @@ def test_passing_trainer_multiple_gpus_raises_error(self):
                 self.model, self.optimizer, self.data_loader, num_epochs=2, cuda_device=[0, 1],
             )
 
+    @pytest.mark.skipif(not torch.cuda.is_available(), reason="No CUDA device registered.")
+    @pytest.mark.skipif(importlib.util.find_spec("apex") is None, reason="Apex is not installed.")
```

Comment:
This `importlib` thing is doing something funny to our CI, can you just change it to how we imported it in the trainer itself?

```
try:
	import apex
accept:
	apex = None

...

@pytest.mark.skipif(apex is None, ...)
```

Reply:
Done!